deployment/helm/values.yaml

```diff
@@ -0,0 +1,138 @@
+# Default values for open-zaak.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+settings:
+  name: openzaak
+  subdomain: openzaak
+  domain: zuid-drecht.nl
+  env: dev
+  module: openzaak.conf.docker
+  secret: 7y82z1%ef)k@f=u-_%ri71lk4=7jy^z5#@6d4*92=t@p0-yk5k
+  dsn:
+  version: latest
+  pullPolicy: Always
+  debug: 1
+
+security:
+  allowedHosts: '*'
+
+email:
+  url: localhost
+  port: 867
+  user: email
+  password: '!changeMe!'
+
+nginx:
+  version: stable
+  maxUpload: 4Gi
```

Comment:
variable defined but not used in the nginx configmap, not sure if that's intentional or forgotten